src/config.ts

```diff
@@ -0,0 +1,65 @@
+const hasDOM = typeof document !== 'undefined' && typeof window !== 'undefined';
+const scriptElement: HTMLScriptElement = hasDOM && document.currentScript;
```

Comment:
```suggestion
const scriptElement: HTMLOrSVGScriptElement | null = document?.currentScript;
```

From my perspective, we get the same result with a bit simpler types, right?